docs/reference/src/advanced/backend_linuxkms.md

```diff
@@ -62,6 +62,26 @@ DP-4 (connected: false)
 
 Setting `SLINT_DRM_OUTPUT` to `DP-3` will render on the second monitor.
 
+To select a specific resolution and refresh rate (mode), set the `SLINT_DRM_MODE` variable. Set it
+to `list` to get a list of available modes. For example the output could look like this:
```

Comment:
the output of what?  This sentence could be improved.
(also you could maybe reduce the size of the output further so it doesn't take too much estate)

Reply:
Good idea - thanks!